apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -44,6 +44,7 @@ type FirewallerAPI struct {
 	accessMachine     common.GetAuthFunc
 	accessModel       common.GetAuthFunc
 	logger            loggo.Logger
+	networkService    NetworkService
```

Comment:
And here. The service is congruent with `State`, so put it in the same place.

Reply:
Done.